ehsm_kms_service/couchdb.js

```diff
@@ -0,0 +1,22 @@
+const nano = require('nano')
+
+const {
+  EHSM_CONFIG_COUCHDB_USERNAME,
+  EHSM_CONFIG_COUCHDB_PASSWORD,
+  EHSM_CONFIG_COUCHDB_SERVER,
+  EHSM_CONFIG_COUCHDB_PORT,
+  EHSM_CONFIG_COUCHDB_DB = 'kms_appid_info',
+} = process.env

```

Comment:
Seems these env vars are predefined.
Will be documented or empty value checked here?

Reply:
yes